packages/flutter_tools/lib/src/linux/linux_extension.dart

```diff
@@ -0,0 +1,234 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I assume in practice this wouldn't be one big file, but would be broken down into pieces more like the current structure?

Reply:
Yes, I don't think there is any reason we would require it to be a single file